src/changelog/2017.md

```diff
@@ -1,13 +1,23 @@
 # March 2017
 
-## Support for Elasticsearch 2.4 and 5.2
+## Elasticsearch 2.4 and 5.2 with support for plugins
 
 Elasticsearch 2.4 and 5.2 are now available.  Both have a number of optional plugins avaialble.  See the [Elasticsearch documentation](/configuration/services/elasticsearch.md) for more information.
 
 ## InfluxDB 1.2
 
 A new service type is available for InfluxDB 1.2, a time-series database.  See the [InfluxDB documentation](/configuration/services/influxdb.md) for more information.
 
+# February 2017
+
+## HHVM 3.15 and 3.18
+
+Two new HHVM versions are now available.
+
+## InfluxDB 1.2
+
+A new service type is available for InfluxDB 1.2, a time-series database.  See the [InfluxDB documentation](/configuration/services/influxdb.md) for more information.
+
```

Comment:
InfluxDB is already listed above for March, which is when it went live.  No need to list it in February, too. :smile: